testcontainers/core/cleaner.py

```diff
@@ -0,0 +1,14 @@
+from docker.errors import NotFound
+
+from testcontainers.core.utils import setup_logger
+
+logger = setup_logger(__name__)
+
+
+def stop_silent(container):
+    try:
+        container.stop()
+    except NotFound:
+        pass
+    except Exception as e:
+        logger.exception(e)
```

Comment:
Please add debug logs here so we'll have visibility on which containers are being shut down because they weren't already.

Also, does this work well with pytest-xdist?